scripts/github-status/github-status.rb

```diff
@@ -117,6 +117,10 @@ def show_forcerebuild_pull_requests
     options[:pr] = pr
   end
 
+  opts.on('-r', '--repository REPO', 'Github Org Repository') do |repo|
```

Comment:
can we do it similar to this?
```ruby
opts.on('-r', '--repository ORG/REPO', 'Github Org Repository') do |repo|
```
the problem is you need to know the format of the `REPO` arg. it needs to look like "org/repo"
or we add another option for the organization... either way would be better IMO